examples/memspace/memspace_hmat.c

```diff
@@ -0,0 +1,133 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
```

Comment:
do you feel like extending the web docs..?

// https://oneapi-src.github.io/unified-memory-framework/examples.html

Reply:
It can be separate PR (in the distant future).

Btw this examples are requested by Igor for a conference/workshop, so maybe after it we can reuse Igor's materials.

Reply:
sounds legit, we could add a simple section with 2 sentences per each new example - just to have a stub for that "distant future" 😉

Reply:
done